src/AdvApi32.Desktop/AdvApi32.cs

```diff
@@ -19,14 +19,169 @@ public static partial class AdvApi32
         /// because services and service groups share the same name space.
         /// </summary>
         public const string SC_GROUP_IDENTIFIER = "+";
-        public const uint INFINITE = 0xFFFFFFFF;
+        public const uint SERVICE_NO_CHANGE = 0xFFFFFFFF;
```

Comment:
@vbfox should we also remove the explicit underlying type in enums (ex. [here](https://github.com/AArnott/pinvoke/blob/master/src/AdvApi32.Desktop/AdvApi32%2BServiceAccess.cs#L17)) and leave the default type (int)?